build/windows/installer/conf/telegraf.conf

```diff
@@ -124,7 +124,7 @@
 #Prometheus Custom Metrics
 [[inputs.prometheus]]
   interval = "$AZMON_TELEGRAF_CUSTOM_PROM_INTERVAL"
-  
+
```

Comment:
Unnecessary change in file

Reply:
> Unnecessary change in file

Yes, this is number of log files to keep and 3 should be fine with each 1MB. other places we have 5 - https://github.com/microsoft/Docker-Provider/blob/ci_prod/kubernetes/linux/main.sh#L940.